assets/custom.css

```diff
@@ -1,11 +1,34 @@
-body {
+body,
+.header__active-menu-item {
     font-family: 'Manrope', sans-serif;
 }
+a {
+    text-decoration: none;
+}
+
+.hover {
+    color: #D87D4A !important;
```

Comment:
you don't want to use `!important` unless you ABSOLUTELY have to. 

Most times, you won't have to. You can use nesting to override other CSS rules 
example:
a.element-class .hover {
color: #d87d4a 
}